libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Login/SFLoginViewController.m

```diff
@@ -244,20 +244,17 @@ - (SFSDKLoginHostListViewController *)createLoginHostListViewController {
 }
 
 - (BOOL)shouldAutorotate {
-    return NO;
+    return YES;
 }
 
 - (UIInterfaceOrientationMask)supportedInterfaceOrientations {
-    return UIInterfaceOrientationMaskPortrait;
+    return UIInterfaceOrientationMaskAll;
 }
 
 - (void)layoutWebView {
     if (nil != _oauthView) {
         [_oauthView removeFromSuperview];
         [self.view addSubview:_oauthView];
-        NSDictionary *views = NSDictionaryOfVariableBindings(_oauthView);
```

Comment:
@trooper2013 removing this fixed the constraint warnings we talked about. I tested on iPad and iPhone and didn't see a difference in the UI without them, let me know if there's anything else I should check